### ISSUE TITLE
Groupings of form controls lack proper labeling

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2250]
This is an issue that appears in multiple places around manifold, much like global issues around misuse of <nav> elements, headings, and single item lists. Collectively, small but recurring issues like this have a significant negative impact on accessibility. I will focus on one specific instance of this issue.
## Steps to Reproduce
1. Go to the the "View" panel in the readers banner
2. Notice that the checkboxes "Yours" and "Others" are visually grouped under the label "HIGHLIGHTS".
3. Examine the HTML and notice that programmatically the checkboxes are grouped in a <div> with no role or aria-lablelby properties
For a better understanding of the impact of this:
1. Navigate to this panel with a screen reader active (NVDA is free and open source (!) for windows https://www.nvaccess.org/download/, on linux, Orca is best for GNOME based DEs https://help.gnome.org/users/orca/stable/)
2. Open the panal and press tab once to put focus on the first checkbox "yours".
3. Notice that the announcement does not provide any useful context for the checkboxes.
## Observed Behavior
Throughout manifold there are groups of checkboxes (and other controls such as radio buttons) that are visually grouped but lack a grouping mechanic that is exposed to assistive technologies like screen readers. For the case study of the View panel in the reader, NVDA announces this when navigating to the first group of checkboxes, "HIGHLIGHTS", from the close button:
```
Navigation Landmark <- this is a problem that has been reported elsewhere
List with three items <-this is the list of groupings of checkboxes
Clickable, "your" checkbox, checked <- the user has no idea what this means without context
Clickable, "others" checkbox, checked <- this is confusing because the last announcement said three items and only two are announced
```
## Expected Behavior
For this case study the expected screen reader announcement would be:
```
List with three items
"highlights" grouping
Clickable, "your" checkbox, checked
Clickable, "others" checkbox, checked
```
It is expected that a grouping of form controls is immediately inside an element that exposes a grouping to assistive technologies. There are several ways to accomplish this, examples can be found here: https://www.w3.org/WAI/GL/wiki/Using_grouping_roles_to_identify_related_form_controls. In our case study this helps the user to understand that there is a list of three groupings of checkboxes, not three checkboxes. Additionally the user understands that "yours" exists within the context of "highlights" without having to use a different screen reader mode to read the static text above the checkboxes.
## Test Environment
NVDA 2019.1.1
ESR Firefox 60.7.0esr
